meta-luneui/recipes-qt/qt5/qtwebengine/0012-screenInfoFromQScreen-fallback-on-primary-screen-if-.patch

```diff
@@ -0,0 +1,30 @@
+From 63c026fd91c5cf03fce41658ae998816867d5c78 Mon Sep 17 00:00:00 2001
+From: Christophe Chapuis <chris.chapuis@gmail.com>
+Date: Wed, 1 Jan 2020 16:04:49 +0000
+Subject: [PATCH] screenInfoFromQScreen: fallback on primary screen if no
+ screen
+
+This way, a correct value can be retrieved for the size of the screen.
+
+Signed-off-by: Christophe Chapuis <chris.chapuis@gmail.com>
+---
+ src/core/render_widget_host_view_qt.cpp | 3 +++
+ 1 file changed, 3 insertions(+)
+
+diff --git a/src/core/render_widget_host_view_qt.cpp b/src/core/render_widget_host_view_qt.cpp
+index 3be0c939..a37f9940 100644
+--- a/src/core/render_widget_host_view_qt.cpp
++++ b/src/core/render_widget_host_view_qt.cpp
+@@ -255,6 +255,9 @@ private:
+ static content::ScreenInfo screenInfoFromQScreen(QScreen *screen)
+ {
+     content::ScreenInfo r;
++    
++    if (!screen) screen = qGuiApp->primaryScreen();
```

Comment:
Would be nice to have the screen = qGuiApp->primaryScreen(); inside of {} like the other if conditions.

Reply:
For one-liners, the rest of the code uses just a indent, no {}.
I can do the change, but I'm not sure the patch would be more clear?

Reply:
I was just thought to always use {} after an if for clarity ;)